packages/plugins/typed-variable-modal/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
You can now use ``../../webpack.config.js``.

Reply:
done